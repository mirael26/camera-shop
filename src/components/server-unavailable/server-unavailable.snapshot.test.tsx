import { renderWithReduxAndRouter } from '../../test/helpers/render-with-redux-and-router';
import ServerUnavailable from './server-unavailable';

test('ServerUnavailable renders correctly', () => {
  const { asFragment } = renderWithReduxAndRouter(<ServerUnavailable/>);
  expect(asFragment()).toMatchSnapshot();
});
