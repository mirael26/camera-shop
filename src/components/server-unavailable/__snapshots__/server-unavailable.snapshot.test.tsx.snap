// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ServerUnavailable renders correctly 1`] = `
<DocumentFragment>
  <section
    class="server-unavailable"
  >
    <div
      class="container"
    >
      <h1
        class="title title--h2 server-unavailable__title"
      >
        –°–µ—Ä–≤–µ—Ä –Ω–µ–¥–æ—Å—Ç—É–ø–µ–Ω üò•
      </h1>
      <h1
        class="server-unavailable__text"
      >
        –ü–æ–∂–∞–ª—É–π—Å—Ç–∞, –ø–æ–ø—Ä–æ–±—É–π—Ç–µ –ø–æ–∑–¥–Ω–µ–µ
      </h1>
      <a
        class="btn btn--purple"
        href="/"
      >
        –ü–µ—Ä–µ–∑–∞–≥—Ä—É–∑–∏—Ç—å
      </a>
    </div>
  </section>
</DocumentFragment>
`;
